log.go

```diff
@@ -39,6 +40,7 @@ func UseLogger(logger btclog.Logger) {
 	blockntfns.UseLogger(logger)
 	pushtx.UseLogger(logger)
 	connmgr.UseLogger(logger)
+	query.UseLogger(logger)
```

Comment:
IMO it's an issue that right now these all use the same logger as it means there's no natmepsacing between the messages. In `lnd`, we can't go `BTCN=debug` without getting the flood of all the peer messages, and there's no way to turn that off. Not blocking for this PR, but just want to not that here as we add yet another hidden sub-logger.

Reply:
Created https://github.com/lightningnetwork/lnd/issues/3662